test/fixtures/categories.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+
+two:
+  name: MyString
```

Comment:
You'll want your test fixtures to have different values, if at some point you use them.